src/engine/ruleset/decoders/f5/f5-afm.yml

```diff
@@ -0,0 +1,85 @@
+---
+name: decoder/f5-afm/0
+
+metadata:
+  module: F5
+  title: F5 AFM Decoder
+  description: Decodes F5 BIG-IP Advanced Firewall Manager logs
+  compatibility: >
+    New Wazuh Engine
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/10
+  references:
+    - TBD
+
+sources:
+  - decoder/queue-syslog/0
+  - decoder/queue-localfile/0
+
+parse:
+  logpar:
+    # iusm modtempo olab6078.home olaboris tur itv [F5@odoco acl_policy_name=ria acl_policy_type=min acl_rule_name=ite action=Closed hostname=tatemac3541.api.corp bigip_mgmt_ip=10.228.193.207 context_name=liqua context_type=ciade date_time=Jan 29 2016 06:09:59 dest_ip=10.125.114.51 dst_geo=umq dest_port=2288 device_product=pexe device_vendor=nes device_version=1.2262 drop_reason=reveri errdefs_msgno=boNemoe errdefs_msg_name=equepor flow_id=eni ip_protocol=ipv6 severity=low partition_name=ehend route_domain=ritquiin sa_translation_pool=umqui sa_translation_type=reeufugi source_ip=10.208.121.85 src_geo=sperna source_port=884 source_user=billoi translated_dest_ip=10.165.201.71 translated_dest_port=6153 translated_ip_protocol=tatemU translated_route_domain=deF translated_source_ip=10.11.196.142 translated_source_port=5222 translated_vlan=iatnu vlan=3810
+    - event.original: <~>F5@<~tmp> <~kv1> date_time=<@timestamp/%b %d %Y %H:%M:%S> <~kv2>
```

Comment:
Improve (Design): You could use a single temporary variable and in the end delete only this one. For example: ~tmp